app/services/registries/create_service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Registries
+  class CreateService < ::Registries::BaseService
+    def execute
+      @registry  = Registry.new(params)
+      @valid     = @registry.valid?
+      @messages  = @registry.errors.messages
+      @reachable = true
+
+      check!
+      if @valid
+        Namespace.update_all(registry_id: @registry.id) if @registry.save
+      end
+
+      @registry
+    end
+
+    protected
+
+    def check!
+      return unless @valid
+      check_uniqueness!
+      return unless @valid
+      check_reachability! unless @force
+    end
+
+    def check_uniqueness!
+      return unless Registry.any?
+      @valid = false
+      @messages[:status] = "You can only create one registry"
```

Comment:
Is `status` the best name? We were using field names but in this case it's not related to a field. Just wondering...

Reply:
I didn't know how to name it... Maybe `uniqueness` ? or `limit` ? or `instances` ?

Reply:
I think `uniqueness` is fine. What do you think?